docker-compose.accounts.yml

```diff
@@ -41,6 +41,8 @@ services:
       - SKYNET_ACCOUNTS_LOG_LEVEL=${SKYNET_ACCOUNTS_LOG_LEVEL}
       - KRATOS_ADDR=${KRATOS_ADDR}
       - OATHKEEPER_ADDR=${OATHKEEPER_ADDR}
+    volumes:
+      - ./docker/accounts/conf:/accounts/conf
```

Comment:
Should be `./docker/accounts/conf:/conf` and then we set the environment var 
`- ACCOUNTS_JWKS_FILE=/conf/jwks.json`

Reply:
What is the motivation behind this? `ACCOUNTS_JWKS_FILE` currently defaults to `/accounts/conf/jwks.json`. We can play around with that any way we like, of course, but I would prefer to use the default value, so if we happen to not set it on some server it would still work as expected.